pipeline/ingest/session_ingest.py

```diff
@@ -7,6 +8,9 @@
 schema = dj.schema(get_schema_name('ingestion'))
 
 
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
```

Comment:
might want to move logging cfg to a .util package/function (eventually?) - i think the mapshell or maybe mlims has the best examples that I know of, not sure what's going on in other projects